blocks/basic/test/qa_Selector.cpp

```diff
@@ -115,140 +122,177 @@ const boost::ut::suite SelectorTest = [] {
         execute_selector_test({.nSamples = 5,                                                   //
             .mapping                     = {{0, 0}, {1, 1}, {2, 2}},                            //
             .inValues                    = {{1}, {2}, {3}},                                     //
             .outValues                   = {{1, 1, 1, 1, 1}, {2, 2, 2, 2, 2}, {3, 3, 3, 3, 3}}, //
-            .monitorSource               = -1U,                                                 //
-            .monitorValues               = {},                                                  //
+            .inTags                      = {{tag1}, {tag2}, {tag3}},                            //
+            .outTags                     = {{tag1}, {tag2}, {tag3}},
+            .monitorSource               = -1U, //
+            .monitorValues               = {},  //
             .backPressure                = false,
-            .ignoreOrder                 = false});
+            .ignoreOrder                 = true});
     };
 
-    "Selector<T> only one input used"_test = [] {
+    "Selector<T> only one input used"_test = [tag1, tag2, tag3] {
         execute_selector_test({.nSamples = 5,                         //
             .mapping                     = {{1, 1}},                  //
             .inValues                    = {{1}, {2}, {3}},           //
             .outValues                   = {{}, {2, 2, 2, 2, 2}, {}}, //
-            .monitorSource               = -1U,                       //
-            .monitorValues               = {},                        //
+            .inTags                      = {{tag1}, {tag2}, {tag3}},  //
+            .outTags                     = {{}, {tag2}, {}},
+            .monitorSource               = -1U, //
+            .monitorValues               = {},  //
             .backPressure                = false,
-            .ignoreOrder                 = false});
+            .ignoreOrder                 = true});
     };
 
-    "Selector<T> all for one"_test = [] {
+    "Selector<T> all for one"_test = [tag1, tag2, tag3] {
+        const Tag newTag1{6, tag1.map};
+        const Tag newTag2{10, tag2.map};
+        const Tag newTag3{13, tag3.map};
         execute_selector_test({.nSamples = 5,                                                       //
             .mapping                     = {{0, 1}, {1, 1}, {2, 1}},                                //
             .inValues                    = {{1}, {2}, {3}},                                         //
-            .outValues                   = {{}, {1, 2, 3, 1, 2, 3, 1, 2, 3, 1, 2, 3, 1, 2, 3}, {}}, //
-            .monitorSource               = -1U,                                                     //
-            .monitorValues               = {},                                                      //
+            .outValues                   = {{}, {1, 2, 2, 3, 3, 3, 1, 1, 1, 1, 2, 2, 2, 3, 3}, {}}, //
```

Comment:
The default scheme should add a
```cpp
 Annotated<bool, "synch combined port", Doc<"true: input ports connected to the same output port are synchronised">> synch_combined_ports = true;
```
and produce
```cpp
.outValues                   = {{}, {1, 2, 3, 1, 2, 3, 1, 2, 3, 1, 2, 3, 1, 2, 3}, {}}, //
```

i.e. having an item-by-item interleaved output.